flutter/test/integrations/more_native_sdk_integration_test.dart

```diff
@@ -0,0 +1,114 @@
+@TestOn('vm')
```

Comment:
I'm not really sure what to do with this test file, since these are different tests than those in `native_sdk_integration_test.dart` and I think they should stay in different files. Any suggestions for a better file name?

Reply:
What about naming one
`native_sdk_integration_test.dart` -> `init_native_sdk_integration_test.dart`
and the other 
`more_native_sdk_integration_test.dart` -> `native_sdk_integration_test.dart`.

Reply:
Ah yes, thanks. I like that